assume/units/dsm_load_shift.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: ASSUME Developers
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import pyomo.environ as pyo
+
+
+class DSMUnit:
```

Comment:
I think you wanted to rename this to something else, right? :) @Manish-Khanra

Reply:
Ah, Yes. Thanks for reminding. :)
Now its renamed.